src/main/java/uk/gov/hmcts/darts/common/entity/CaseEntity.java

```diff
@@ -14,76 +13,98 @@
 import jakarta.persistence.Table;
 import lombok.Getter;
 import lombok.Setter;
-import org.hibernate.annotations.Type;
+import org.apache.commons.collections4.CollectionUtils;
 
 import java.time.OffsetDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
 @Entity
-@Table(name = "court_case")
+@Table(name = CaseEntity.TABLE_NAME)
 @SuppressWarnings({"PMD.ShortClassName"})
 @Getter
 @Setter
 public class CaseEntity {
 
```

Comment:
Been wondering whether we should change this to CourtCaseEntity to align with the table name?

Reply:
hmm, probably should yeah, i'll make the change